### PR TITLE
feat: added option 'useJsonWireFormat'

### PR DESCRIPTION
**Pull request by oyvindwe**
Fixes issue #571

### CODE REVIEW

**Comment by stephenh** [comment]
@oyvindwe hm, manually re-running didn't work, but I merged a potential fix to use `GITHUB_TOKEN`; can you try pulling that in and see if that works?

I think I'd recently bumped the version of `actions/checkout` and yours might be the first PR running on the new `v3`.

**Comment by oyvindwe** [comment]
@stephenh Merged - ready for workflow approval again.

**Review by stephenh — APPROVED** [review]

File: src/enums.ts [hunk 1]
```diff
@@ -27,19 +27,20 @@ export function generateEnum(
   }
 
   const delimiter = options.enumsAsLiterals ? ':' : '=';
+  const stringEnums = options.stringEnums || (options.onlyTypes && options.useJsonWireFormat);
```
[the remark this answers is not in the document]
Comment (oyvindwe) [reply src/enums.ts:30]: Yes, that's a good idea. I can fix this in a separate PR. (I just pushed this to this branch, as I didn't notice this PR is already merged.)

**Pull request merged**